index.js

```diff
@@ -15,13 +15,13 @@ class Client {
       throw new TypeError('Please provide auth options.')
     }
 
-    const { email, ethAddress, apiKey, password } = auth
+    const { email, ethAddress, apiKey, password, userId } = auth
```

Comment:
Do we really need to add `userId` to the `auth` object? If the intended usage is instantiating the client without parameters, IMO we should not expect the user providing a `userId`, is that correct?